app/mailers/learning_hours_mailer.rb

```diff
@@ -0,0 +1,24 @@
+class LearningHoursMailer < ApplicationMailer
+  def learning_hours_report_email(user)
+    # user is either an Admin or Supervisor, this mailer is invoked through the rake task :monthly_learning_hours_report.rake
+
+    @user = user
```

Comment:
Are you using `@user` in the mailer at all? If not, this doesn't need to be an instance variable, and you can just use `user` everywhere in this method.

Reply:
Yeah I am using `@user` in a couple of places throughout the mailer. (: